### DISCUSSION TITLE
Great work

### DISCUSSION THREAD

**Discussion by Onfroygmx** [marlonrichert/zsh-launchpad#2] [Show and tell]
Hello, 

Very Nice defaults and especially great job on the comments!!!

1 issue I noticed within the completion system:
It picks up the first option in alphabetical order and not giving the choice,
ex:
`nano .config/zsh/<tab>`
gives => `nano .config/zsh/functions/`

and then 
`nano .config/zsh/functions/<tab>`
gives: `nano .config/zsh/functions/extract-images`

Unless I type: `nano .config/zsh/rc.d/07<tab>`

I won't get that option.
I would expect to chose between the files and folders, because I don't always want the first in alphabetical order.

And again great work on the comments, because understandable basic configs are really missing today in the ZSH world.

Onf

**Comment by marlonrichert** [comment]
Thank you for your kind praise! 🤗

About the completion problem: Please open an issue for that in the [`zsh-autocomplete` repo](https://github.com/marlonrichert/zsh-autocomplete).

**Reply by Onfroygmx** [reply]
Do you need a full bug report? If yes, which specific information do you want? => Not sure to have the time today.
Do you just want a reference to work on it later? => I can  do that today.

Probably just a:
zstyle ':completion:*'              menu select=2

Missing somewhere.

**Reply by marlonrichert** [reply]
Yeah, I do need a full bug report, as I am unable to reproduce it. And if I cannot reproduce it, I cannot fix it. But no hurry; just do it when you have time. 🙂

**Reply by marlonrichert** [reply]
@Onfroygmx PS: I maintain all of my software for Zsh in my free time. If you enjoy using it, please consider giving back by [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
